src/Service/DocumentUploader.php

```diff
@@ -38,6 +39,41 @@ public function __construct(SluggerInterface $slugger, string $uploadDocumentDir
         $this->mimeTypeGuesser = $mimeTypeGuesser;
     }
 
+    /**
+     * Creates and returns new document from filename.
+     */
+    public function createDocumentFromPath(string $fileName, User $user, string $documentName, string $documentType): Document
+    {
+        $document = new Document();
+
+        $document->setFilename($fileName);
+        $document->setDocumentName($documentName);
+        $document->setPath($this->getFilepathFromProjectDirectory($fileName));
+        $document->setUploadedBy($user);
+        $document->setType($documentType);
```

Comment:
Should original file name be set here?

Reply:
Documents created directly via `createDocumentFromPath` all come from a template and therefore don't have an original file name. I will try to make this clearer.